lib/capistrano/tasks/cachetool.rake

```diff
@@ -0,0 +1,11 @@
+namespace :cachetool do
+  namespace :opcache do
+    task :reset do
+      on release_roles(fetch(:cachetool_roles)) do
+        within fetch(:cachetool_working_dir) do
+          execute :cachetool, "opcache:reset --web --web-path=#{release_path}/web --web-url=#{fetch(:cachetool_reset_web_url)}", raise_on_non_zero_exit: false
```

Comment:
Lately, I used `--web=SymfonyHttpClient --web-basic-auth=user:secretpassword` to clear the cache of website protected by Basic HTTP Auth